cmd/cmd_controller.js

```diff
@@ -433,6 +433,20 @@ class EmbarkController {
     fs.copySync(embarkConfig, dappConfig);
     console.log(__('webpack config ejected to: ').dim.yellow);
     console.log(`${dappConfig}`.green);
+    var dappOverrides = fs.dappPath('webpack-overrides.js');
+    var embarkOverrides = fs.embarkPath('lib/pipeline', 'webpack-overrides.js');
+    ext = 1;
+    let dappOverridesOld = dappOverrides;
+    while (fs.existsSync(dappOverridesOld)) {
```

Comment:
Why are you using `while` instead of `if` ?

Reply:
Seems like he's looking for the biggest override number as he increments the number inside the loops.
Though, the name seems weird. Wouldn't it be : `path/webpack-overrides.js.1`? Shouldn't it be `path/webpack-overrides.1.js;` ?

Reply:
@jrainville is correct re: the biggest number.

The rename for overrides works like the one I implemented previously for config:

```
dappConfig + `.${ext}`
dappOverrides + `.${ext}`
```

If it seems better to have the filenames end with `js.${ext}`, I can make that change for both.

Reply:
Done.